src/main/java/seedu/address/model/person/Module.java

```diff
@@ -30,6 +32,7 @@
      */
     public Module(String module) {
         requireNonNull(module);
+        module = module.toUpperCase(Locale.ROOT);
```

Comment:
I think this line can be removed as the parser will always send a capitalised module code into the variable module